crates/env/src/engine/on_chain/ext.rs

```diff
@@ -76,6 +77,8 @@ define_error_codes! {
      /// The call to `debug_message` had no effect because debug message
     /// recording was disabled.
     LoggingDisabled = 9,
+    /// The call dispatched by `call_runtime` was executed but returned an error.
+    CallRuntimeReturnedError = 10,
```

Comment:
please roll it back to `CallRuntimeFailed`, not to mess with ` #[allow(clippy::enum_variant_names)]`

I'll rename it in the pallet as well

Reply:
name reverted; allowance removed